design/ceph/ceph-nfs-ganesha.md

```diff
@@ -129,6 +130,13 @@ spec:
             configMap:
               name: rook-ceph-nfs-organization-sssd-config
               defaultMode: 0600 # mode must be 0600
+        sssdGenericFiles:
```

Comment:
Since it's not specifically `sssd.conf`, I think `supplementalFiles`, `additionalFiles`, `genericFiles`, or something like that is sufficient.

```suggestion
        supplementalFiles:
```